src/views/router_views/extension-reason.njk

```diff
@@ -69,6 +68,7 @@
                   {% endif %}
              {{ govukButton({
                 text: i18n.submit_request,
+                  href: "/persons-with-significant-control-extension/first-extension-request-successful" +"?companyNumber="+ companyNumber + "&"+ "selectedPscId="+ selectedPscId +"%3F", 
```

Comment:
This may also be something for a later story, but ought every page Post back to the same handler for validation and then it directs to the next handler.
eg [executePost in Confirm Company](https://github.com/companieshouse/psc-verification-web/blob/33dea3d747ae5ff93f39ce28e4775be7565956a8/src/routers/handlers/confirm-company/confirmCompany.ts#L55-L59)